jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -496,12 +465,12 @@ String getFormat() {
    */
   Optional<Path> getExtraDirectoryPath() {
     // TODO: Should inform user about nonexistent directory if using custom directory.
-    if (System.getProperty(PropertyNames.EXTRA_DIRECTORY_PATH) != null) {
-      return Optional.of(Paths.get(System.getProperty(PropertyNames.EXTRA_DIRECTORY_PATH)));
-    }
-    return extraDirectory.path == null
-        ? Optional.empty()
-        : Optional.of(extraDirectory.path.toPath());
+    String property = getProperty(PropertyNames.EXTRA_DIRECTORY_PATH);
+    return property == null
```

Comment:
Might consider rewriting this as `if/else` for readability.

Reply:
Sorry, I only meant one instance, but I think it's fine. Feel free to change the others back to `? :` if you liked that.